code/lib/cli/src/doctor/index.ts

```diff
@@ -105,6 +105,15 @@ export const doctor = async ({
 
   const allDependencies = (await packageManager.getAllDependencies()) as Record<string, string>;
 
+  if (!('storybook' in allDependencies)) {
+    logDiagnostic(
+      'Package storybook not found',
```

Comment:
Maybe put `storybook` in quotes, or chalk it?

Reply:
<img width="654" alt="image" src="https://github.com/user-attachments/assets/7165fa3d-3142-4879-a476-f935e01a9cef">
like this?